### PR TITLE
feat: minor ui updates

### PR DESCRIPTION
**Pull request by sakul-budhathoki**
Fixes:-
https://github.com/berty/bugs/issues/131
https://github.com/berty/bugs/issues/83
https://github.com/berty/bugs/issues/130
https://github.com/berty/bugs/issues/127
https://github.com/berty/bugs/issues/134

### CODE REVIEW

File: js/packages/components/chat/message/UserMessage.tsx [hunk 1]
```diff
@@ -149,6 +149,7 @@ export const UserMessage: React.FC<{
 		cmd,
 	} = getUserMessageState(inte, members, convKind, previousMessage, nextMessage, border, color)
 
+	console.log(previousMessage?.medias?.[0]?.mimeType)
```
Comment (clegirar) [inline js/packages/components/chat/message/UserMessage.tsx:152]: I don't think this log was necessary, else code LGTM 👍

**Review by clegirar — APPROVED** [review]
code LGTM 👍

File: js/packages/components/chat/message/UserMessage.tsx (hunk 1 above)
Comment (sakul-budhathoki) [reply js/packages/components/chat/message/UserMessage.tsx:152]: Thanks I will fix it.

**Comment by zxxma** [comment]
LGTM 👍  :)

**Pull request merged**